tgui/packages/tgui/interfaces/TechFab.js

```diff
@@ -3,7 +3,7 @@ import { Stack, Collapsible, Tooltip, Icon, Box, Button, LabeledList, Input, Sec
 import { Window } from '../layouts';
 import { Fragment } from 'inferno';
 import { capitalize, createSearch } from 'common/string';
-import { sendLogEntry } from 'tgui-dev-server/link/client';
+import { sendLogEntry } from 'tgui-dev-server/link/client.cjs';
```

Comment:
This should probably be removed, seems unused, looks like a leftover from testing.

Reply:
Leaving this for potential cleanup for later

Reply:
There are a couple of those from when Frost originally ported it all. There are also some other unused imports here and there, but eh it doesn't really hurt anything to be there. We could probably do a cleanup pass later, but that should be separate I think